actions.py

```diff
@@ -10,6 +10,18 @@
 from humor_commands import *
 from constants import *
 
+def covid_standard_time():
+    """
+    Prints the number of days since 1 March 2020. With thanks to https://github.com/brasstax/Silva.
+    """
+    now = datetime.datetime.now(datetime.timezone.utc)
+    delta = datetime.now(datetime.timezone.utc).date() - datetime.date(2020, 3, 1)
```

Comment:
How about now.date()?
The datetime module is weird, and this needs another datetime before the datetime.now because of the way I imported it.

Reply:
Done.